lib/rubocop/cop/rspec/factory_bot/attribute_defined_statically.rb

```diff
@@ -31,12 +31,15 @@ class AttributeDefinedStatically < Cop
             (send _ !#reserved_method? $...)
           PATTERN
 
-          def_node_search :factory_attributes, <<-PATTERN
+          def_node_matcher :factory_attributes, <<-PATTERN
             (block (send _ #attribute_defining_method? ...) _ { (begin $...) $(send ...) } )
           PATTERN
 
           def on_block(node)
-            factory_attributes(node).to_a.flatten.each do |attribute|
+            attributes = factory_attributes(node) || []
+            attributes = [attributes] unless attributes.is_a?(Array)
```

Comment:
Nitpick: I _think_ these two lines could be written a bit simpler:
```rb
attributes = Array(factory_attributes(node))
```

Reply:
I already suggested this, but unfortunately, node deconstructs itself in this case
https://github.com/rubocop-hq/rubocop-rspec/pull/949#discussion_r446888166

Reply:
Ah, thanks for clarifying.